apps/sms/views/conversation/style/message.css

```diff
@@ -49,6 +49,7 @@
 
 .message p.message-content-body {
   white-space: pre-wrap;
+  font-family: "Firefox Emoji", sans-serif;
```

Comment:
Don't we want to make Emoji font as a CSS variable and put it somewhere in a shared place so that other apps can use the same and it can be easily updated in the future (gaia-theme maybe)?